doc/README.md

```diff
@@ -0,0 +1,143 @@
+documentation.
+
+The following example programs may be of interest:
+
+- [examples/monte-carlo.rs](https://github.com/rust-lang-nursery/rand/blob/master/examples/monte-carlo.rs)
+- [examples/monty-hall.rs](https://github.com/rust-lang-nursery/rand/blob/master/examples/monty-hall.rs)
+
+## References
+
+API documentation can be found:
+
+- [`rand` API on docs.rs](https://docs.rs/rand/)
+- [`rand_core` API on docs.rs](https://docs.rs/rand_core/)
+- [self-published API on github.io](https://rust-lang-nursery.github.io/rand/rand/index.html) (latest code in master branch)
+- by running `cargo doc --no-deps --all --all-features`
+
+## Project policies
+
+### Open Participation
+
+This project is open to contributions from anyone, with the main criteria of
+review being correctness, utility, project scope, and good documentation. Where
+correctness is less obvious (PRNGs and some type-conversion algorithms),
+additional criteria apply (see below).
+
+Additionally we welcome feedback in the form of bug reports, feature requests
+(preferably with motivation and consideration for the scope of the project),
+code reviews, and input on current topics of discussion.
+
+Since we must sometimes reject new features in order to limit the project's
+scope, you may wish to ask first before writing a new feature.
+
+### Stability and Portability
+
+We try to follow [semver rules](https://docs.npmjs.com/misc/semver) regarding
+API-breaking changes and `MAJOR.MINOR.PATCH` versions:
+
+-   New *patch* versions should not include API-breaking changes or major new
+    features
+-   Before 1.0, *minor* versions may include API breaking changes. After 1.0
+    they should not.
+-   We may make pre-releases like `0.5.0-pre.0`. In this case:
+    
+    -   although these are public versions, they are not used by default unless
+        opting into using a pre-release on the specific `MAJOR.MINOR.PATCH`
+        version
+    -   pre-releases are considered semantically less than their stem (e.g.
+        Cargo may automatically upgrade from `0.5.0-pre.0` to `0.5.0`)
+    -   all pre-release versions are unstable and may make any change
+    -   we make no commitment to support users of pre-releases
+
+Additionally, we must also consider *value-breaking changes* and *portability*.
+A function is *value-stable* if, given the same inputs:
+
+-   it is portable (produces the same results on all platforms)
+-   changing the output value for some input in a new library version is
+    considered a breaking change
+
+Note that some Rand functionality is supposed to be value stable, and some
+functionality is supposed to be non-deterministic (i.e. depend on something
+external). Some functionality may be deterministic but not value-stable.
+
+A trait should define which of its functions are expected to be value-stable.
+A type (e.g. a PRNG) is value-stable if all implementations of trait functions
+for that type are value-stable where expected to be.
```

Comment:
You wrote things careful here, maybe with the distinction between `from_seed` and `from_rng` in mind? I had to read it a couple of times to completely understand. Maybe:

Both traits and types (such as a PRNG or distribution) should declare whether they are value-stable.
A trait should define for which of its methods it expects the implementations to be value-stable.
When a type that implements such a trait declares it is value-stable, it must keep up these promises.
As an example, `SeedableRng::from_seed` is required to be value-stable, but `SeedableRng::from_rng` not. RNGs implementing the trait are value-stable when they guarantee `SeedableRng::from_seed` is value-stable, while `SeedableRng::from_rng` may receive optimisations.